library/card/card.css

```diff
@@ -203,4 +203,169 @@
 
 /*  Card Component Ends -- Basic */
 
+
+/* Card Component Starts -- Prod Horizontal Card */
+
+.dui-card-prod-hzntl {
+    width: fit-content;
+    height: auto;
+
+    background: var(--dui-light-theme-bg);
+    display: grid;
+
+    grid-auto-columns: 4fr 4fr 3fr 3fr 3fr 3fr;
+    grid-template-areas:
```

Comment:
Suggestion  - 

grid-auto-columns: 2fr 1fr;

grid-template-areas: 
 'cardImg  cardInfo'
 'cardImg  cardDesc '
 'cardImg  cardActions ';
 
 Won't this help you ?